### ISSUE TITLE
Consuming a simple ONNX model for text classification

### ISSUE THREAD

**Issue by oliverw** [dotnet/machinelearning#4239]
I'm trying to consume a trained scikit learn model that was converted to ONNX. Attempting to load the model with:

```c#
var pipeline = ctx.Transforms.ApplyOnnxModel(
    new[] { "label", "probabilities" }, new[] { "input" }, "test.onnx");
```

Results in the following exception:

> System.ArgumentOutOfRangeException: 'dim (Parameter 'Dimension { } in ONNX tensor cannot exceed the maximum of 32-bit signed integer.')
> Actual value was 0.'
> 

**Update:**

Saving the model with `target_opset = 9` results in:

> [ErrorCode:Fail] Load model from ..\..\..\..\..\Data\MLModel\test.onnx failed:Fatal error: StringNormalizer is not a registered function/op

**Model creation in python:**

```python
import os
from sklearn.pipeline import Pipeline
from sklearn.feature_extraction.text import TfidfVectorizer
from sklearn.svm import LinearSVC
from skl2onnx import convert_sklearn
from skl2onnx.common.data_types import StringTensorType
import pandas as pd
from onnxmltools import save_model

data = pd.read_csv('training_data.csv', sep='\t')

pipeline = Pipeline([
  ('tfidf', TfidfVectorizer()),
  ('clf', LinearSVC()),
])

pipeline.fit(data["Keywords"], data["Label"])

onnx = convert_sklearn(pipeline, name='test_predictor',
  initial_types=[('input', StringTensorType([1, 1]))])

save_model(onnx, "test.onnx")

```

**Netron:**

![image](https://user-images.githubusercontent.com/51688/65391096-6fe19180-dd65-11e9-965a-3eab334533a6.png)

**Comment by KsenijaS** [comment]
Can you please share your training_data.csv file

**Comment by KsenijaS** [comment]
@oliverw Closing the issue for now, we will reopen the issue if the missing file is provided.

**Comment by oliverw** [comment]
@KsenijaS Here you go (excerpt)

[training_data.zip](https://github.com/dotnet/machinelearning/files/3698685/training_data.zip)

**Comment by KsenijaS** [comment]
Hi,
You're getting an exception when using ctx.Transforms.ApplyOnnxModel because your onnx model is inccorect. Label output should not be empty. I don't know why your onnx model is incorrect, there may be an issue with convert_sklearn, but I'm not sure.

When you try to save model to opset 9, you are getting an exception because operator StringNormalizer does not exist in opset 9. StringNormalizer is introduced in opset 10.
Please see https://github.com/onnx/onnx/blob/master/docs/Operators.md#StringNormalizer for more information.
